### PR TITLE
Add gas limit to mew link

### PR DESCRIPTION
**Pull request by TomSesselmann**

### CODE REVIEW

**Comment by TomSesselmann** [comment]
I've added gasLimit to the `CandidateEthTx` model but I'm not sure where I'm supposed to add the `"0x030d40"` value. Do I just add it to the `initModel`?

File: web/src/SecureVote/Eth/Utils.elm [hunk 1]
```diff
@@ -29,8 +29,11 @@ processCandidateTx candTx =
 
         data =
             candTx.data
+
+        gasLimit =
+            candTx.gasLimit
     in
-    Maybe.map4 MinEthTx from to data (intToHexEth value)
+    Maybe.map5 MinEthTx from to data gasLimit (intToHexEth value)
```
Comment (TomSesselmann) [inline web/src/SecureVote/Eth/Utils.elm]: Also, do we need to do `(fromHex gasLimit)` here? @XertroV

File: web/src/SecureVote/Eth/Models.elm [hunk 2]
```diff
@@ -6,16 +6,19 @@ type alias CandidateEthTx =
     , value : Int
     , to : Maybe String
     , data : Maybe String
+    , gasLimit : Maybe String
```
Comment (XertroV) [inline web/src/SecureVote/Eth/Models.elm]: this should be `gas` not `gasLimit` https://github.com/ethereum/wiki/wiki/JavaScript-API#contract-methods

Comment (XertroV) [reply web/src/SecureVote/Eth/Models.elm]: Also, just set to a string - it's constant

File: web/src/SecureVote/Eth/Models.elm [hunk 3]
```diff
@@ -6,16 +6,19 @@ type alias CandidateEthTx =
...
     }
 
 
+nullCandidateEthTx : CandidateEthTx
 nullCandidateEthTx =
-    CandidateEthTx Nothing 0 Nothing (Just "")
+    CandidateEthTx Nothing 0 Nothing (Just "") Nothing
```
Comment (XertroV) [inline web/src/SecureVote/Eth/Models.elm]: Set this to the String

File: web/src/SecureVote/Eth/Utils.elm (hunk 1 above)
Comment (XertroV) [reply web/src/SecureVote/Eth/Utils.elm]: No

**Comment by TomSesselmann** [comment]
changes made @XertroV

**Pull request merged**